portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,14 +19,36 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-
+  String [] comments = {
```

Comment:
This can be `private static final String[] COMMENTS = {`

[Constants use `CONSTANT_CASE`](https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names)

In general, limit access as much as possible, and make all static variables final. I'll see if I can find resources I can provide to you all with tips like this.

Reply:
awesome will do!

Reply:
done